governance/process.md

```diff
@@ -1,15 +1,74 @@
+Committee][TOC], the work of the group often originates from group members with
+ideas on how to reduce risk in cloud native applications in alignment with the
+charter of the group and of interest and importance to the community.
 
-This document explains how we transform ideas from our community into
-projects with a defined set of deliverables
-or a team to focus on a larger stream of work that may involve one or more
-projects and activities.
+This document explains how we transform ideas from our community into projects
+with a defined set of deliverables or a team to focus on a larger stream of work
+that may involve one or more projects and activities.
+
+## Getting on the Agenda
+
+In order to better plan and facilitate meetings, Security TAG has **three** ways
+in which a topic may be added to the [Agenda for planned
+meetings](https://docs.google.com/document/d/170y5biX9k95hYRwprITprG6Mc9xD5glVn-4mB2Jmi2g/edit#heading=h.q3539ohya47f)
+
+### Presentations
+
+Presentations require a [presentation
+issue](https://github.com/cncf/tag-security/issues/new?assignees=&labels=usecase-presentation%2C+triage-required&template=presentation.md&title=%5BPresentation%5D+Presentation+Title)
+to be created.  Please fill this in as completely and accurately as you can,
+providing multiple dates for presentations is ideal.
+
+Once a presentation issue is submitted a Security TAG representative will be
+assigned to or review the issue to triage the request. Once triaged, the
+Security TAG representative will perform due diligence on the issue to ensure it
+adhere's to our requirements for presentation content. If the [requirements are
+met](presentations.md), the Security TAG representative will then add the
+topic, link the issue, and provide the point of contact and themselves on the
+Agenda.
+
+We strive to have no more than two presentations in a given meeting, but prefer
+only one to ensure we have ample time for lively discussions.
+
+### Topics and updates
+
+We expect our members working on projects and members of groups to periodically
+provide updates to the TAG as part of our longstanding meeting format. If you
+are currently working on a project or a member of a group, your updates are
+already embedded in the format. If they are not, reach out to
+[#tag-security-governance](https://cloud-native.slack.com/archives/C0230RW8V2T)
+to have them added.
+
+If you have a topic or point of discussion you which to raise which is not:
+
+* already part of an update
+* not a proposal or suggestion
+
+but, as part of a working session is:
+
+* seeking input
+* probably no more than 15 minutes of discussion
+* adhere's to our governance and code of conduct
+
+please add it to the Agenda for a WORKING SESSION with a `[PENDING]` prefix on
+the topic, put your name next to it, and let us know by dropping note in the
+[#tag-security-governance](https://cloud-native.slack.com/archives/C0230RW8V2T) channel.
+We will provide feedback if necessary.
```

Comment:
think an example here would be appropriate something like `[PENDING] Discuss on new awesome cloud native security idea - R. Racoon`